amaranth/sim/pysim.py

```diff
@@ -331,11 +332,12 @@ def write(self, addr, value, mask=None):
 
 class _PySimulation(BaseSimulation):
     def __init__(self):
-        self.timeline  = _Timeline()
-        self.signals   = SignalDict()
-        self.memories  = {}
-        self.slots     = []
-        self.pending   = set()
+        self.timeline     = _Timeline()
+        self.signals      = SignalDict()
+        self.memories     = {}
+        self.slots        = []
+        self.pending      = set()
+        self.print_output = sys.stdout
```

Comment:
(For cursed reasons; also happy with `self.stdout`)

```suggestion
        self.output = sys.stdout
```